package.json

```diff
@@ -30,13 +30,13 @@
   },
   "dependencies": {
     "ethers": "^5.7.2",
-    "forta-agent": "^0.1.26",
     "lru-cache": "^8.0.4"
   },
   "devDependencies": {
     "@types/jest": "^27.0.1",
     "@types/nodemon": "^1.19.0",
     "forta-agent-tools": "^3.2.7",
+    "forta-agent": "^0.1.26",
```

Comment:
why did you move `forta-agent` into `devDependencies`? 🤔 the packages inside devDep are not installed into the bot's dockerized image. so the bot wouldn't be able to function if it doesn't install `forta-agent`